### PR TITLE
Regionalized Logging configuration

### LINKED ISSUE #1742: Regionalized Logs to europe-west1
## Summary

Enable support for Regionalized Logs and limit location to `europe-west1`. We will change this to `europe-west6` once they enable support for that region.

https://cloud.google.com/logging/docs/regionalized-logs

## Checklist:

- [x] Searched the [existing issues](https://github.com/WorldHealthOrganization/app/issues) to ensure you are not creating a duplicate.
- [x] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md).

### PR DESCRIPTION
**Pull request by matthewblain**
Enable Regionalized Logging Configuration for default logs. Fixes #1742.

See https://cloud.google.com/logging/docs/regionalized-logs

Unless overridden, this will configure the logs at europe-west1, as the closest available option for the default europe-west6 application location.

Note: This will cause users to run into https://github.com/hashicorp/terraform-provider-google/issues/7811 , updated readme to express workaround.

## Screenshots

<details>
<summary>Screenshots</summary>
Add any relevant before/after screenshots here
</details>

## How did you test the change?

- [ ] iOS Simulator
- [ ] iOS Device
- [ ] Android Simulator
- [ ] Android Device
- [ ] `curl` to a dev App Engine server
- [x] other, please describe

Ran on my local configuration. Todo: test on one of the other environments.

## Checklist:

- [ ] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md) and verified that all contributions are properly licensed pursuant to the [LICENSE](https://github.com/WorldHealthOrganization/app/blob/master/LICENSE).

### CODE REVIEW

**Comment by brunobowden** [comment]
@matthewblain - you can append "gitignore" to the name of a directory or file (before extension) and it will ignored.

Both examples:
```
server/terraform/mblain-gitignore/main.tf

server/terraform/mblain/main-gitignore.tf
```

File: server/terraform/modules/myhealth/variables.tf [hunk 1]
```diff
@@ -18,6 +18,12 @@ variable "region" {
   default = "europe-west6"
 }
 
+# Logs buckets are not available in all regions, so may vary from "region". 
+variable "logs_region" {
+  type    = string
+  default = "europe-west1"
```
Comment (brunobowden) [inline server/terraform/modules/myhealth/variables.tf:24]:
```suggestion
  # TODO: move to Switzerland: https://github.com/WorldHealthOrganization/app/issues/1754
  default = "europe-west1"  # Belgium
```

**Review by brunobowden — APPROVED** [review]

File: server/terraform/modules/myhealth/main.tf [hunk 2]
```diff
@@ -85,6 +85,46 @@ resource "google_app_engine_application" "gae" {
...
+# Redirects App Engine and other non-audit logs to a regional bucket.
+# 
+# Note: Terraform does not currently handle _Default buckets well. It will fail
+# on the first apply; you must then 'import' the sink into the tf state with:
+# terraform import module.myhealth.google_logging_project_sink.default projects/${var.project_id}/sinks/_Default
+# See: https://github.com/hashicorp/terraform-provider-google/issues/7811
+resource "google_logging_project_sink" "default" {
+  name = "_Default"
+
+  destination = "logging.googleapis.com/projects/${var.project_id}/locations/${var.logs_region}/buckets/${var.logs_region}-logs-bucket"
```
Comment (brunobowden) [inline server/terraform/modules/myhealth/main.tf:112]: See if this has a `self_link` field that gives you what you need:
```
terraform state show module.myhealth.google_logging_project_bucket_config.regional_log_bucket.self_link
```
If it does, then switch over with this:
```suggestion
  destination = google_logging_project_bucket_config.regional_log_bucket.self_link
```

**Review by brunobowden — APPROVED** [review]

File: server/terraform/modules/myhealth/main.tf (hunk 2 above)
Comment (matthewblain) [reply server/terraform/modules/myhealth/main.tf]: Ah, I tried to dos this before but failed somehow. Much better, as I've now fixed an error elsewhere in the config!

**Pull request merged**